fe2-android/app/src/main/java/com/github/dedis/popstellar/model/objects/Lao.java

```diff
@@ -408,6 +431,18 @@ public void setElections(Map<String, Election> elections) {
     this.elections = elections;
   }
 
+  /**
+   * Class which return the last roll call open
+   *
+   * @return Rollcall the roll call with the last ending tim e
+   */
+  public RollCall lastRollCallClosed() throws NoRollCallException {
+    return this.getRollCalls().values().stream()
+            //.filter(RollCall::isClosed)
```

Comment:
As #1116 was merged, you can merge master in and uncomment this.

Reply:
Yes, I forgot to uncomment it, sorry 😅